book/text.md

```diff
@@ -39,6 +39,23 @@ typing. Variations—like bold or italic letters—were called that type's
 
 [^2]: The word is related to *foundry*, which would create the little
     metal shapes.
+    
+::: center
```

Comment:
```suggestion
::: {.center}
```

Reply:
Actually this works but you're right, we typically do this this way.